jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/parser/WindowRateControl.java

```diff
@@ -39,6 +39,11 @@
     private final int maxEvents;
     private final long window;
 
+    public static WindowRateControl ofSecond(int maxEvents)
```

Comment:
Use `long` instead of `int` to comply with the `java.time` API that all use `long` for time values.

Reply:
Doh! it's `maxEvents` not time.

Reply:
@lachlan-roberts I was wrong in suggesting `ofSeconds()` then.

Perhaps `fromRate(int maxEvents)`?